packages/components/src/core-badge/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { privateApis } from '@wordpress/components';
```

Comment:
This is a duplication of an existing calypso component right? Should we extract this to its own PR and update the existing usage of the current component to avoid the unnecessary duplication in the code base?

Reply:
Yeah, I mention this in the description. I was planning on doing it in a follow up, but can do it before.

Reply:
https://github.com/Automattic/wp-calypso/pull/103069